addon/components/base/bs-contextual-help.js

```diff
@@ -213,20 +211,20 @@ export default Component.extend(TransitionSupport, {
   arrowElement: null,
 
   /**
-   * The jQuery object of the viewport element.
+   * The DOM element of the viewport element.
    *
    * @property viewportElement
    * @type object
    * @readonly
    * @private
    */
   viewportElement: computed('viewportSelector', function() {
-    return $(this.get('viewportSelector'));
+    return document.getElementById(this.get('viewportSelector'));
```

Comment:
Oh, I think that was by mistake. Should really use `querySelector` as well. Good catch!

- [ ] Fix https://github.com/kaliber5/ember-bootstrap/pull/326#discussion-diff-116896205R222